contracts/AelinUpFrontDeal.sol

```diff
@@ -618,20 +634,24 @@ contract AelinUpFrontDeal is AelinERC20, MinimalProxyFactory, IAelinUpFrontDeal
         _;
     }
 
-    modifier blockTransfer() {
-        require(false, "cant transfer deal tokens");
-        _;
-    }
-
-    function transfer(address _dst, uint256 _amount) public virtual override blockTransfer returns (bool) {
-        return super.transfer(_dst, _amount);
+    function transferVestingShare(
```

Comment:
I've changed the logic here so we don't have to claim before transferring a share of a vesting schedule

Reply:
maybe I do need to add an event in `_createVestingToken` to track which schedules are being created. Thoughts?

Reply:
I will take a closer look at this in a bit. not sure yet